runtime/src/lib.rs

```diff
@@ -347,16 +354,6 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl sp_consensus_aura::AuraApi<Block, AuraId> for Runtime {
-		fn slot_duration() -> u64 {
-			Aura::slot_duration()
-		}
-
-		fn authorities() -> Vec<AuraId> {
-			Aura::authorities()
-		}
-	}
-
 	impl sp_session::SessionKeys<Block> for Runtime {
 		fn generate_session_keys(seed: Option<Vec<u8>>) -> Vec<u8> {
 			opaque::SessionKeys::generate(seed)
```

Comment:
I think you can remove these impl as well. 

At this point, we need to think through the UX of this workshop. The deploy & demo steps currently are for the PoS default. It's kind of confusing to still be able to run a PoS node and see blocks still getting finalized, even though this PR makes this a PoW implementation already. 

Which version do we want to be the workshop? If it's pow, we could disable the PoS features so people shouldn't be able to run a node for it. 

Either way, we should still update the readme to reflect the new UX/demo/deployment instructions.

Reply:
All the PoS stuff is removed in this PR. Running `utxo-workshop --dev` starts a mining node which mines PoW blocks.